msf_updates/pose_sensor_fix.yaml

```diff
@@ -1,33 +1,72 @@
+data_playback: true             # set to true when playback, false on the real system
 
-data_playback: true
+##############################
+#########IMU PARAMETERS#######
+##############################
+# The imu measurement model used in msf contains two types of sensor errors, a fast additive white noise and 
+# a slower varying sensor bias.  
+# See the following link for more information 
+# https://github.com/ethz-asl/kalibr/wiki/IMU-Noise-Model-and-Intrinsics
+# 
+# The white noise is characterized with the continous time noise spectral density. 
+# The noise spectral density is sometime also refered to as noise density.
+# The units of the noise spectral density are:
+#  acc:  [m/s^2/sqrt(Hz)]
+#  gyro: [rad/s/sqrt(Hz)]
+# The noise spectral density can be found in the datasheet.
+# 
+# The variation of the bias is characterized as a random walk. 
+# Again see https://github.com/ethz-asl/kalibr/wiki/IMU-Noise-Model-and-Intrinsics for more information
+# The units of the random walk are:
+#  acc:  [m/s^3/sqrt(Hz)]
+#  gyro: [rad/s^2/sqrt(Hz)]
+ 
+####### ADIS
```

Comment:
@simonlynen  this is from your config file, I dont know where these values come from